dietpi-install.sh

```diff
@@ -16,6 +16,15 @@ touch "/etc/pve/qemu-server/$ID.conf"
 # Get the storage name from the user
 STORAGE=$(whiptail --inputbox 'Enter the storage name where the image should be imported:' 8 78 --title 'DietPi Installation' 3>&1 1>&2 2>&3)
 
+# Ask if user wants to use BTRFS 
+read -p "Use BTRFS storage? (y/n) " use_btrfs
```

Comment:
As I got confused myself about the aim of the issue/PR:
```suggestion
read -p "Is it a BTRFS storage? (y/N) " use_btrfs
```
Otherwise people could get the idea that they can choose to use Btrfs here, or even for the guest internal filesystem, while at this point the Proxmox storage needs to be a Btrfs volume already.